padinfo/padinfo.py

```diff
@@ -524,22 +524,33 @@ async def collabscroll(self, ctx, *, query: str):
 
     @commands.command(aliases=['leaders', 'leaderskills', 'ls'])
     @checks.bot_has_permissions(embed_links=True)
-    async def leaderskill(self, ctx, left_query: str, right_query: str = None, *, bad=None):
+    async def leaderskill(self, ctx, *, whole_query=''):
         """Display the multiplier and leaderskills for two monsters
 
         If either your left or right query contains spaces, wrap in quotes.
         e.g.: [p]leaderskill "r sonia" "b sonia"
         """
-        if bad:
-            await ctx.send(inline('Too many inputs. Try wrapping your queries in quotes.'))
-            return
+        # order of separators is '/' > '"' > ','
+        pref_query = []
+        if '/' in whole_query:
+            left_query, right_query = whole_query.split('/')[:2]
```

Comment:
one fancy thing you can do is
`left_query, right_query, *bad = whole_query.split('/')`
just to weed out bad input but this is totally optional. 

I would recommend doing that on the space one though.